### ISSUE TITLE
wifi connection and redirection after connecting

### ISSUE THREAD

**Issue by csheldon93** [Spritetm/esphttpd#18]
After loading esphttpd firmware, everything works really well, except two (most important things):

1) Redirection after connecting to ESP Access point. It redirectes but never gets loaded, and I have to push reload button on chrome browser multiple times until it loads the index page
2) Scanning of available wifi networks...after switching to STA+AP mode, it never finishes scanning and never shows available networks...it just sits there saying scanning...

any help please?

**Comment by Cicero-MF** [comment]
This happens to me as well.  Android Samsung S8, Chrome browser, captive portal doesn't work well.  It is almost like it gets overwhelmed by connections...but I haven't looked into it enough to confirm that theory.  As a temporary workaround I added a whitelist of URL's it allows autoredirects to index, and ignore the rest of the URL's.

**Comment by mharizanov** [comment]
Same here. I believe proper captive portal is much more complex than responding with the local IP. See this discussion for example https://unix.stackexchange.com/questions/432190/why-isnt-androids-captive-portal-detection-triggering-a-browser-window

**Comment by Cicero-MF** [comment]
Yes, true, I also follow this project with similar issues https://github.com/idolpx/mobile-rr/issues/2#issuecomment-432749682
Although as far as I understand, esphttpd doesn't exactly have an automatic captive portal which is meant to open a browser window automatically with all the problems associated with that (checking for internet connected sites etc), but it is still meant to redirect any input URL to the index page - Sprite called it captive DNS.  But even the redirect is flaky and requires refreshing.

**Comment by wdouglas1999** [comment]
Hi, I am using esp-idf with ESP32. I have the issue of scanning of available WiFi networks which never seems to appear. 
Is there any work around or solution I can try for this? 
Thank you in advance.
